pkg/runtime/runtime.go

```diff
@@ -650,10 +650,6 @@ func (a *DaprRuntime) initRuntime(ctx context.Context) error {
 		return err
 	}
 
-	if err := a.processor.Subscriber().InitProgramaticSubscriptions(ctx); err != nil {
```

Comment:
Is this called somewhere else?

Reply:
Yes, here https://github.com/dapr/dapr/blob/7ea7016c0150e6a7b7bc7c30e34d56f0001c5e90/pkg/runtime/processor/subscriber/subscriber.go#L242